All Core Devs Meetings/Meeting 74.md

```diff
@@ -0,0 +1,349 @@
+
+**Tim Beiko:** For phase 2, within the next two weeks, mid-November.
+
+**Danno Ferrin:** We'll have it out next week. 
+
+**Hudson Jameson:** The Ethereum Foundation and/or Ethereum Cat Herders are publishing a blog on the block number and what software to upgrade around when most clients update their download links.
+
+
+Video: [[10:24]](https://youtu.be/aZ0S_oLSwhE?t=624)
+
+**Danno Ferrin:** Should we formally make 1671 accepted?
+
+- [EIP 1679](https://eips.ethereum.org/EIPS/eip-1679 )
+
+**Hudson Jameson:** That sounds good.
+
+**James Hancock:** EIPs are to be moved to accepted and final for integration in Istanbul which has begun for the client. The included EIPs:
+- EIP 152
+- EIP 1108
+- EIP 1344
+- EIP 1884
+- EIP 2028
+- EIP 2200
+
+
+**Hudson Jameson:** I second that. We should probably have motions.
+
+**Danno Ferrin:** Agreed.
+
+
+## 2. Berlin
+
+Links: [Berlin Meta EIP](https://eips.ethereum.org/EIPS/eip-2070)
+
+
+### 2.1 Ice Age
+
+Video: [[7:55]](https://youtu.be/aZ0S_oLSwhE)
+
+**Tim Beiko:** A couple calls ago we said that the Ice Age would start kicking in next summer, please correct me if I'm wrong. We probably want an EIP in Berlin that kicks back the Ice Age.
+
+**Hudson Jameson:** James Hancock decided to write that. Not a huge rush. We delay it the same time each time.
+
+**Danno Ferrin:** That gives us a little over a year each time.
+
+
+### 2.2 Tentatively Accepted EIPs
+
+
+#### 2.2.1 EIP-663
+[**Unlimited SWAP and DUP instructions**](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-663.md )
+
+Video: [[44:51]](https://youtu.be/aZ0S_oLSwhE?t=2691)
+
+**Hudson Jameson:**  So there has been a lot of comments in the Ethereum Magicians forum on this.
+
+- [Ethereum Magicians Thread](https://ethereum-magicians.org/t/eip-663-unlimited-swap-and-dup-instructions/3346)
+
+
+**Greg:** I don't think SWAP DUP should be here without basic decisions on the Spec. I don't consider it blessed, but there's not large mountain of work needed.
+
+**Alex Beregszaszi:** Blessed means no objection from Core Devs for the idea. I don't believe any of the Core Devs objected on the idea.
+
+**Greg:** Ok. I still don't think an EIP should come to us without consensus in other discussion that it is a design which will work. It looked to me that it wasn't ready, and there was disagreement among the community, including some Core Devs.
+
+#### 2.2.2 EIP-1380
+[**Reduced gas cost for call to self**](https://eips.ethereum.org/EIPS/eip-1380)
+
+Video: [[56:00]](https://youtu.be/aZ0S_oLSwhE?t=3358)
+
+**Alex Beregszaszi:** Benchmarks show some reduction can be made, but not to the degree of the original proposal. 
+
+#### 2.2.3 EIP-1702
+[**Generalized account versioning scheme**](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1702.md)
+
+Video: [[34:57]](https://youtu.be/aZ0S_oLSwhE?t=2097)
+
+**James Hancock:** Several EIPs are gated by account versioning.
+
+**Wie Tang:** New specification for account versioning made. 
```

Comment:
Please change all those lines to `Wei Tang`. That's my name!

Reply:
Sorry, didn't notice the misspelling. The attendance list had the correct spelling, but it seemed I wrote it wrong once in the notes, and copied it over several times. It's fixed now